pulp_2to3_migration/app/plugin/content.py

```diff
@@ -310,8 +312,17 @@ def get_remote_by_importer_id(importer_id):
 
                         lces = pulp2lazycatalog.filter(pulp2_storage_path=image_path)
                         if lces:
+                            # To create multiple remote artifacts, create multiple instances of
+                            # declarative content which will differ by url/remote in their
+                            # declarative artifacts
+                            at_least_one_lce_migrated = False
+
                             for lce in lces:
                                 remote = get_remote_by_importer_id(lce.pulp2_importer_id)
+
+                                if not remote and not downloaded:
+                                    continue
```

Comment:
@goosemania I'm not 100% sure this is the correct fix?  I just copied the general strategy being used a couple lines down.

Reply:
it seems to be a reasonable change. however i cannot wrap my head around - can this logic lead to a half migrated unit because of multiple remotes and some of them can be missing?

Reply:
I'm attempting to avoid that with line 330 and 339

Reply:
@dralley , +1 to approach, seems reasonable to me.
@ipanova , I think even if some remotes are missing, it's not a big issue if we have at least one successful lce migration.

Reply:
sounds good, thanks for confirmation